src/components/Involved.jsx

```diff
@@ -6,7 +6,7 @@ const Involved = () => {
   return (
     <div>
       <Title text="Get Involved" />
-      <div className="flex justify-content-center pb-40 pt-12">
+      <div className="grid md:flex md:flex-row space-y-5 grid-cols-2 justify-content-center pb-40 pt-12">
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/97c9b313-c697-4330-9193-abb77c40aea6)

space-y causes the social media components to be misaligned. instead use gap for tailwind grid

https://tailwindcss.com/docs/gap